packages/verification/src/models/v1/flashcall-verification-report-response/flashcall-verification-report-response.ts

```diff
@@ -1,5 +1,4 @@
-import { VerificationReportResponsePrice } from '../verification-report-response-price';
-import { VerificationStatusEnum } from '../enums';
+import { ReasonEnum, SourceEnum, VerificationStatusEnum } from '../enums';
 import { Identity } from '../identity';
 
 export interface FlashCallVerificationReportResponse {
```

Comment:
I'm not finding associated response from online doc synchronized with this declaration 
- start verification: https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verifications/#tag/Verifications/operation/StartVerification!c=200&path=1/id&t=response
- report a verification: https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verifications/#tag/Verifications/operation/ReportVerificationByIdentity!c=200&path=1/id&t=response
- get verificaiton by id: https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verification-status/#tag/Verification-status/operation/VerificationStatusById!c=200&path=1/id&t=response

Reply:
The schema is defined as a oneOf in the spec: https://gitlab.com/sinch/sinch-projects/enterprise-and-messaging/documentation/developer-experience/newdocs/-/blob/development/docs/verification/api-reference/verification.yaml?ref_type=heads#L895
But I've removed less properties than what is in the current doc, still following the discussion with the backend team